packages/block-library/src/navigation/edit.js

```diff
@@ -276,6 +276,7 @@ function Navigation( {
 							__experimentalPassedProps={ {
 								className: 'wp-block-navigation__container',
 							} }
+							templateLock={ false }
```

Comment:
So that the block appender doesn't appear underneath the Navigation block in `nav-menus.php`, I added `templateLock: 'all'` to the Block Editor's settings. Doing this, however, causes the block appender to not appear within the Navigation block. This is because the `InnerBlocks` here inherits its parent's template lock setting.

To fix this, I set `templateLock={ false }` on the Navigation block's `InnerBlocks`. This prevents the setting from being inherited. In my opinion this is correct because blocks like Navigation and Social Links are entirely useless if one cannot add or remove their inner blocks. Inserting and removing inner blocks is the _primary functionality_ of blocks like these.

Reply:
Got it!